integration/linux/build/Dockerfile-centos.template

```
@@ -20,7 +20,9 @@ ENV GCC_VERSION 11
 # baseurl. Switch to the archive.kernel.org mirror while we are at it.
 RUN set -ex \
 	&& ulimit -n 1024 \
+%%IF VARIANT=7%%
 	&& sed -i 's/enabled=1/enabled=0/g' /etc/yum/pluginconf.d/fastestmirror.conf \
+%%ENDIF%%
```

Better not to rely on its presence at all:

```suggestion
	&& ( [ -e /etc/yum/pluginconf.d/fastestmirror.conf ] && sed -i 's/enabled=1/enabled=0/g' /etc/yum/pluginconf.d/fastestmirror.conf || true ) \
```